src/lando/headless_api/admin.py

```diff
@@ -24,18 +22,13 @@ class AutomationActionJobInline(ReadOnlyInline):
     _target_object = "actions"
     readonly_fields = ("action_type", "data", "order")
 
-    def formfield_for_dbfield(self, db_field: models.Field, **kwargs) -> forms.Field:
-        """
-        Forbid alteration of the JobAction list.
-        """
-        formfield = super(ReadOnlyInline, self).formfield_for_dbfield(
-            db_field, **kwargs
-        )
-        if db_field.name == "actions":
-            formfield.widget.can_add_related = False
-            formfield.widget.can_change_related = False
-            formfield.widget.can_delete_related = False
-        return formfield
+    def has_add_permission(self, request, obj=None) -> bool:  # noqa: ANN001
```

Comment:
Are these changes to the headless API admin view related to the landing job views? Maybe they could be sent in a separate PR.

Reply:
Only discovered when testing.  https://github.com/mozilla-conduit/lando/pull/417